src/integration-test/integration/MappingsClient.test.ts

```diff
@@ -0,0 +1,537 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as chaiAsPromised from "chai-as-promised";
+import { expect, use } from "chai";
+import { CalculatedPropertyCreate, CalculatedPropertyType, CalculatedPropertyUpdate, CustomCalculationCreate, CustomCalculationUpdate, DataType, ECProperty, GroupCreate, GroupPropertyCreate, GroupPropertyUpdate, GroupUpdate, MappingCopy, MappingCreate, MappingsClient, MappingUpdate, QuantityType } from "../../reporting";
+import "reflect-metadata";
+import {accessToken, testIModel, testIModelGroup } from "../utils/";
```

Comment:
Inconsistent `import` formatting - missing space between `{` and `accessToken`.

Reply:
This was fixed before but only now committed